tests/alive-tv/calls/musttail-memcpy.srctgt.ll

```diff
@@ -0,0 +1,18 @@
+define tailcc i64 @src(i64 noundef %arg) {
+  %call2 = alloca i64
+  %call3 = alloca i64
+  store i64 %arg, ptr %call2, align 8
+  musttail call tailcc void @llvm.memcpy.p0.p0.i64(ptr byval(ptr) %call3, ptr byval(ptr) %call2, i64 8, i1 false)
```

Comment:
this not a good test because it's mixing several conditions at once

Reply:
Separated it in multiple functions.